src/bluesky/bundlers.py

```diff
@@ -427,6 +427,13 @@ def emit_event(readings: Optional[dict[str, Reading]] = None, *args, **kwargs):
                     f"{readable_obj} has async read() method and the callback "
                     "passed to subscribe() was not called with Dict[str, Reading]"
                 )
+            assert (
```

Comment:
why are we adding asserts in the main code base?  I see there is one above here as well.